### PR TITLE
Create temurin-vdr-generator repo

### PR DESCRIPTION
**Pull request by smlambert**
A new repo to contain the Vulnerability Disclosure Report generation scripts (related: https://github.com/adoptium/temurin/issues/1 and https://github.com/adoptium/adoptium/issues/209)

### CODE REVIEW

**Comment by netomi** [comment]
to which project shall this repo belong? temurin?

File: otterdog/adoptium.jsonnet [hunk 1]
```diff
@@ -725,6 +724,17 @@ orgs.newOrg('adoptium') {
...
       ],
     },
+    orgs.newRepo('temurin-vdr-generator') {
+      allow_auto_merge: true,
+      default_branch: "main",
+      description: "Scripts for generating Vulnerability Disclosure Reports",
+      topics+: [
+        "secure-dev"
+      ],
+      branch_protection_rules: [
```
Comment (gdams) [inline otterdog/adoptium.jsonnet:734]: also not sure if this is set by default CC @netomi

File: otterdog/adoptium.jsonnet [hunk 2]
```diff
@@ -725,6 +724,17 @@ orgs.newOrg('adoptium') {
         },
       ],
     },
+    orgs.newRepo('temurin-vdr-generator') {
+      allow_auto_merge: true,
+      default_branch: "main",
```
[the remark this answers is not in the document]
Comment (netomi) [reply otterdog/adoptium.jsonnet:729]: this can be removed yes

File: otterdog/adoptium.jsonnet (hunk 1 above)
Comment (netomi) [reply otterdog/adoptium.jsonnet:734]: there is no default branch protection rule setup by default.

For the vert.x project we did something like that: 

https://github.com/eclipse-vertx/.eclipsefdn/blob/main/otterdog/eclipse-vertx.jsonnet#L18

**Comment by netomi** [comment]
permissions are setup.

**Pull request merged**